src/content/structured/get-started/testing-components.mdx

```diff
@@ -0,0 +1,131 @@
+---
+path: "/get-started/install-components/testing"
+
+navPriority: 7
+
+date: "2022-11-15"
+
+title: "Testing with Jest"
+
+subTitle: "To use the components follow the relevant framework instructions."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/testing.mdx"
+---
+
+[Jest](https://jestjs.io/) is a JavaScript testing framework designed to ensure correctness of any JavaScript codebase. It allows you to write tests with an approachable, familiar and feature-rich API that gives you results quickly.
+
+These instructions have been written with JavaScript in mind. Jest also supports TypeScript. Further instructions on how to set up Jest for TypeScript can be found on their ['Getting Started'](https://jestjs.io/docs/getting-started#using-typescript) page
+
+## Installing Jest
+
+### Step one
+
+Install Jest with `npm` or `yarn`.
+
+```shell
+npm install --save-dev jest
+
+yarn add --dev jest
+```
+
+### Step two
+
+Create a test file using the extension: `.test.js` and add some tests.
+
+### Step three
+
+Add the following section to your package.json:
+
+```shell
+{
+  "scripts": {
+    "test": "jest"
+  }
+}
+```
+
+### Step four
+
+Finally, run your tests using `npm test` or `yarn test`.
+
+More information on setting up Jest can be found on [Jest’s Getting Started page](https://jestjs.io/docs/getting-started).
+
+## What is the shadow DOM?
+
+A key part of our packages is the shadow DOM. With the shadow DOM, we can encapsulate each component’s markup structure, styling and behaviour so different parts don’t clash.
+
+According to [Mozilla](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM), the “shadow DOM allows hidden DOM trees to be attached to elements in the regular DOM tree — this shadow DOM tree starts with a shadow root, underneath which you can attach any element, in the same way as the normal DOM.”
+
+## Testing shadow components
+
+In the DOM, elements can be found using `document.querySelector`. Shadow components can be found in a similar way using the `shadowRoot` prefix. The shadow root is the root node of the shadow DOM tree.
+
+Note: components that having been slotted in can be found in DOM. They are not a part of their parent component's shadow DOM
```

Comment:
```suggestion
Note: components that have been slotted in can be found in the 'light' DOM. They are not a part of their parent component's shadow DOM
```

Reply:
Had a quick look for a website we could refer to for a definition of the 'light' DOM'. Perhaps reference this: https://web.dev/shadowdom-v1/#terminology-light-dom-vs-shadow-dom